databricks/koalas/indexes.py

```diff
@@ -1066,7 +1064,7 @@ def drop(self, labels):
         >>> index.drop([1])
         Int64Index([2, 3], dtype='int64')
         """
-        if not isinstance(labels, (tuple, list)):
+        if not is_list_like(labels):  # TODO: tuple?
```

Comment:
I think maybe we don't need to consider `tuple` here because `Index` cannot have `tuple` value in Koalas anyway ??

```python
>>> ks.Index([(1, 2), 3])  # Cannot have a tuple for Index as mixed type
Traceback (most recent call last):
...
pyarrow.lib.ArrowInvalid: Could not convert (1, 2) with type tuple: did not recognize Python value type when inferring an Arrow data type

>>> ks.Index([(1, 2), (3, 4)])  # This is not gonna be an Index but MultiIndex
MultiIndex([(1, 2),
            (3, 4)],
           )
```

Oh, but if we had any plan for supporting tuple for values, I think It's okay to remain the TODO as it is.